### PR TITLE
Latest News updated for Cosmos

### PR DESCRIPTION
**Pull request by lbliii**
# What does this PR do ?

Add news for Cosmos and previous news/cosmos blurb written by jennifer in a previously closed PR

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# GitHub Actions CI

The Jenkins CI system has been replaced by GitHub Actions self-hosted runners.

The GitHub Actions CI will run automatically when the "Run CICD" label is added to the PR.
To re-run CI remove and add the label again.
To run CI on an untrusted fork, a NeMo user with write access must first click "Approve and run".

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [x] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,33 @@
...
+        </a> (2025-01-07)
+      </summary>
+        The NeMo Framework now supports training and customizing the <a href="https://github.com/NVIDIA/Cosmos">NVIDIA Cosmos</a> collection of world foundation models. Cosmos leverages advanced text-to-world generation techniques to create fluid, coherent video content from natural language prompts.
+        <br><br>
+        You can also now accelerate your video processing step using the <a href="https://developer.nvidia.com/nemo-curator-video-processing-early-access">NeMo Curator</a> library, which provides optimized video processing and captioning features that can deliver up to 89x faster video processing when compared to an unoptimized CPU pipeline.
+      <br><br>
+    </details>
+</details>
+<details open>
+  <summary><b>Generative AI Models</b></summary>
```
Comment (shashank3959) [inline README.md:36]: All models are Generative AI Models. Better to move this news to LLM and MM section

Comment (lbliii) [reply README.md]: Sounds good. I dragged it down to under that section.

File: README.md [hunk 2]
```diff
@@ -165,6 +192,19 @@ Overall, these enhancements make NeMo 2.0 a powerful, scalable, and user-friendl
```
Comment (jgerh) [inline README.md:192]: suggested addition

## Get Started with Cosmos

Cosmos world foundation models are open and available on [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/cosmos/collections/cosmos) and [Hugging Face](https://huggingface.co/collections/nvidia/cosmos-6751e884dc10e013a0a0d8e6). Developers can also run Cosmos world foundation models on the [NVIDIA API Catalog](https://build.nvidia.com/explore/simulation). Also available on the [API catalog](https://build.nvidia.com/nim?filters=usecase%3Ausecase_sdg) are Cosmos tools to enhance text prompts for accuracy, an in-built watermarking system that enables easy future identification of AI-generated sequences, and a specialized model to decode video sequences for augmented reality applications. To learn more, watch the [demo](https://www.youtube.com/watch?v=9Uch931cDx8).

In addition to Cosmos world foundation models, the platform also includes:

- [NeMo Curator](https://developer.nvidia.com/nemo-curator/) for efficient video data curation
- [Cosmos Tokenizer](https://developer.nvidia.com/blog/state-of-the-art-multimodal-generative-ai-model-development-with-nvidia-nemo/) for efficient, compact, and high-fidelity video tokenization
- [NeMo Framework](https://docs.nvidia.com/nemo-framework/user-guide/latest/overview.html) for model training and optimization

[NeMo Curator](https://developer.nvidia.com/nemo-curator#section-demo) for accelerated data processing pipelines is available as a managed service and SDK. Developers can now apply for early access. Cosmos tokenizers are open neural networks available on [GitHub](https://github.com/NVIDIA/Cosmos-Tokenizer) and [Hugging Face](https://huggingface.co/collections/nvidia/cosmos-tokenizer-672b93023add81b66a8ff8e6). 
To get started with NVIDIA Cosmos, see [NVIDIA Cosmos Developers](https://developer.nvidia.com/cosmos) for more information

**Review by jgerh — COMMENTED** [review]
Added a few suggested revisions

File: README.md [hunk 3]
```diff
@@ -155,6 +181,30 @@ Overall, these enhancements make NeMo 2.0 a powerful, scalable, and user-friendl
...
+
+The Cosmos Tokenizer is available now on the <a href=http://github.com/NVIDIA/cosmos-tokenizer/NVIDIA/cosmos-tokenizer>NVIDIA/cosmos-tokenizer</a> GitHub repo and on <a href=https://huggingface.co/nvidia/Cosmos-Tokenizer-CV8x8x8>Hugging Face</a>.
+
+### Get Started with Cosmos
+
+Cosmos world foundation models are open and available on [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/cosmos/collections/cosmos) and [Hugging Face](https://huggingface.co/collections/nvidia/cosmos-6751e884dc10e013a0a0d8e6). Developers can also run Cosmos world foundation models on the [NVIDIA® API Catalog](https://build.nvidia.com/explore/simulation). Also available on the [API catalog](https://build.nvidia.com/nim?filters=usecase%3Ausecase_sdg) are Cosmos tools to enhance text prompts for accuracy, an in-built watermarking system that enables easy future identification of AI-generated sequences, and a specialized model to decode video sequences for augmented reality applications. To learn more, watch the [demo](https://www.youtube.com/watch?v=9Uch931cDx8).
```
Comment (shashank3959) [inline README.md:198]: Can we focus this for NeMo FW since this is the FW github?

Comment (jgerh) [reply README.md:198]: Of course. Do we want to make any statements about NFW and Cosmos in this ReadMe? "Cosmos leverages NeMo for key tasks such as efficient video data curation with NeMo Curator and training and fine-tuning world foundation models, providing the underlying technology for developing advanced physical AI systems. For more information, please see [Cosmos](https://github.com/NVIDIA/Cosmos)."

Comment (lbliii) [reply README.md]: removed!

Comment (shashank3959) [reply README.md]: Sorry I wasn't clear enough!

I did not mean to remove the whole section, but rather shorten it to reflect just the NeMo FW support :) Something like what Jennifer commented above 

### Get Started with Cosmos

NeMo Curator and NeMo Framework support video curation and post-training of the [Cosmos World Foundation Models](https://nvidianews.nvidia.com/news/nvidia-launches-cosmos-world-foundation-model-platform-to-accelerate-physical-ai-development), which are open and available on [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/cosmos/collections/cosmos) and [Hugging Face](https://huggingface.co/collections/nvidia/cosmos-6751e884dc10e013a0a0d8e6). For more information refer to
[video curator docs], and [cosmos post-training with nemo docs]

Comment (shashank3959) [reply README.md]: For NeMo Curator, I think we should link to https://developer.nvidia.com/nemo-curator which has information on video modalities. LGTM otherwise, thank you.

File: README.md [hunk 4]
```diff
@@ -163,6 +189,8 @@ Overall, these enhancements make NeMo 2.0 a powerful, scalable, and user-friendl
 - For an in-depth exploration of the main features of NeMo 2.0, see the [Feature Guide](https://docs.nvidia.com/nemo-framework/user-guide/latest/nemo-2.0/features/index.html#feature-guide).
 - To transition from NeMo 1.0 to 2.0, see the [Migration Guide](https://docs.nvidia.com/nemo-framework/user-guide/latest/nemo-2.0/migration/index.html#migration-guide) for step-by-step instructions.
 
+
```
Comment (jgerh) [inline README.md:192]: ### Get Started with Cosmos

NeMo Curator and NeMo Framework support video curation and post-training of the Cosmos World Foundation Models, which are open and available on [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/cosmos/collections/cosmos) and [Hugging Face](https://huggingface.co/collections/nvidia/cosmos-6751e884dc10e013a0a0d8e6). For more information on video datasets, refer to [NeMo Curator](https://developer.nvidia.com/nemo-curator). To post-train World Foundation Models using the NeMo Framework for your custom physical AI tasks, see the [Cosmos Diffusion models](https://github.com/NVIDIA/Cosmos/blob/main/cosmos1/models/diffusion/nemo/post_training/README.md) and the [Cosmos Autoregressive models](https://github.com/NVIDIA/Cosmos/blob/main/cosmos1/models/autoregressive/nemo/post_training/README.md).

**Comment by jgerh** [comment]
Reviewed latest updates. LGTM.

**Review by shashank3959 — APPROVED** [review]

**Review by ethanhe42 — APPROVED** [review]
lgtm

**Review by invisiblepancake — COMMENTED** [review]
Summary of Changes*
New Sections Added*
Cosmos World Foundation Models Support*

A new section detailing the support for training and customizing the NVIDIA Cosmos collection of world foundation models.
Links to relevant resources and blogs.
Highlights the ability to accelerate video processing using the NeMo Curator library.

Generative AI Models:

New Sections 

A new section discussing significant enhancements to the NeMo platform, focusing on multimodal generative AI models.
Mentions the NeMo Curator and Cosmos tokenizer for efficient data curation and visual data quality improvement.
Provides links to relevant blogs and GitHub repositories.
Cosmos Diffusion and Autoregressive (AR) Models:


Describes advancements in visual foundation models with Cosmos diffusion and AR models.
Details the capabilities and benefits of both model types.
Emphasizes the efficient tokenization process and its impact on generative AI applications.
Directs users to the Cosmos Tokenizer GitHub repo and Hugging Face.



Impact of Changes:

Documentation Enhancement: The new sections provide detailed information on recent advancements and support for new models, helping users stay informed about the latest capabilities of the NeMo framework.
Resource Links: The added links to blogs, GitHub repositories, and Hugging Face resources make it easier for users to access additional information and tools.
User Guidance: The detailed descriptions and summaries guide users on how to leverage the new features and models effectively.
Overall, these changes significantly enhance the README.md file by providing comprehensive information on new features, models, and resources, making it a valuable resource for users of the NeMo framework.

hope this makes sence somewhere =) ty copilot for review help
